recipes/asl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "asl" %}
```

Comment:
@vitaut, thanks for the input. Would it be possible to cut out non-libasl targets from this recipe? Although I suppose it would be silly to do that just for naming concerns. 

I think the main concern is that homebrew already refers to the same package as `asl`. From `homebrew-science/asl.rb`:
``` ruby
class Asl < Formula
  desc "The AMPL modeling language solver library"
  homepage "http://www.ampl.com"
  url "https://github.com/ampl/mp/archive/3.1.0.tar.gz"
```
How hard it is to rename homebrew packages? would that cause huge headaches for dependencies which reference it?